orca/orca-retrofit/orca-retrofit.gradle

```diff
@@ -36,6 +36,8 @@ dependencies {
   testImplementation("org.springframework.boot:spring-boot-test")
   testImplementation("com.squareup.okhttp3:mockwebserver")
   testImplementation("com.squareup.retrofit2:converter-jackson")
+  testImplementation("com.github.tomakehurst:wiremock-jre8")
```

Comment:
removing this since standalone is enough: https://github.com/spinnaker/spinnaker/pull/7085/commits/9501dbdc61e8562bbc45195318aae220f06b7e9a